material-github.user.css

```diff
@@ -1080,8 +1080,282 @@
 	.business-sso .business-sso-panel, .business-sso .org-sso-panel, .org-sso .business-sso-panel, .org-sso .org-sso-panel {
 		background-color: var(--bg1) !important;
 	}
+
 	.review-comment .is-comment-editing {
 	    background-color: var(--bg1);
-	    border: var(--bg1);
+		border: var(--bg1);
+	}
+	
+	/* Header */
+	.Header-link {
+		color: rgba(255, 255, 255, 0.6) !important;

```

Comment:
could you please set it to `color: rgba(255, 255, 255, calc(0.6 * /*[[text_brightness]]*/)) !important;`
This would make it compatible with text brightness selector.